bindings/rust/s2n-tls/src/config.rs

```diff
@@ -277,6 +278,12 @@ impl Builder {
         Ok(self)
     }
 
+    /// Associate a `certificate` and corresponding `private_key` with a config.
+    /// Using this method, at most one config per auth type (ECDSA, RSA, RSA-PSS)
+    /// can be loaded.
```

Comment:
I think my error here was just a simply typo, and it should read
> at most one *certificate* per auth type

Because customers can actually use multiple auth types for each config.